projects/media-viewer/src/lib/annotations/annotation-set/annotation-set.component.ts

```diff
@@ -81,19 +82,22 @@ export class AnnotationSetComponent implements OnInit, OnDestroy {
   }
 
   createBookmark(rectangle: Rectangle) {
-    this.store.pipe(select(fromBookmarks.getBookmarkInfo), take(1))
-      .subscribe((bookmarkInfo) => {
-        const selection = window.getSelection().toString();
+    const selection = window.getSelection().toString();
+    this.store.pipe(select(fromAnnotations.getDocumentIdSetId))
```

Comment:
you can get the documentId from a document selector rather than annotations, so as not to tie this to annotations